server.js

```diff
@@ -5,16 +5,24 @@ const session = require("express-session");
 const MongoStore = require("connect-mongo");
 const router = require("./routes/api.routes");
 const passport = require("./middlewares/passport.middleware");
+const compression = require('compression');
+const helmet = require("helmet");
 
 const app = express();
 
+console.log("server.js -- app initializied");
```

Comment:
We added these debugging `console.log()` statements and commented out the two `app.use(...)` statements during Monday, April 25th's debugging session. Don't forget to remove and uncomment these at some point.